src/quickpicks/schemaRegistries.ts

```diff
@@ -43,6 +43,7 @@ export async function schemaRegistryQuickPickWithViewProgress(): Promise<
  */
 export async function schemaRegistryQuickPick(
   defaultRegistryId: string | undefined = undefined,
+  title?: string,
```

Comment:
Kinda weird that we set this as option, but right above it we present registry id as required but with an undefined default?

Reply:
True, will be cleaning this (and other quickpick) modules up in a follow-on to help with test coverage as well